Files/Interacts/Interaction.cs

```diff
@@ -870,6 +870,24 @@ public async Task<bool> RenameFileItemAsync(ListedItem item, string oldName, str
             return false;
         }
 
+        public bool SetHiddenAttributeItems(ListedItem item, bool isHidden)
```

Comment:
I believe this can be simplified to:
```cs
public void SetHiddenAttributeItems(ListedItem item, bool isHidden)
{
	item.IsHiddenItem = isHidden;
	AssociatedInstance.ContentPage.ResetItemOpacity();
}
```

Reply:
Yes, that's indeed a huge simplification ;-)
I updated it accordingly... Deletion of hidden files and folders also seems to work (but I think that was not me fixing this behavior)